cluster/addons/dns/coredns.yaml.base

```diff
@@ -114,9 +118,6 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
-        - containerPort: 9153
```

Comment:
Why are you eliminating the metrics port being exposed?

Reply:
The exposure of metrics on the service was raised as a security concern in (coredns/deployment#13).  It's my understanding that Prometheus scrapes metrics from pods, not from services - in this way it gathers's regular interval metrics from each pod.  If Prometheus  scraped via the service, it would get a metric from one arbitrary replica each interval, resulting in incomplete/diluted data.  Per my interpretation of k8s docs, and by observation, pods by default expose all ports that their containers expose - and that the ports listed in deployment the spec are only needed when exposing the ports on the service.